website/content/docs/secrets/pki/troubleshooting-acme.mdx

```diff
@@ -0,0 +1,198 @@
+---
+layout: docs
+page_title: 'PKI - Secrets Engine: Troubleshooting ACME'
+description: The PKI secrets engine for Vault generates TLS certificates.
+---
+
+# PKI Secrets Engine - Troubleshooting ACME
+
+Below are suggestions for troubleshooting issues with the ACME clients
+requesting access from Vault. Different clients will behave in different
+ways and have different ways of viewing more details about failures.
+
+## Contacting support
```

Comment:
Hmm, I wanted a very visible directive that says, hey, if you do contact support, be sure to include this extra info you aren't used to providing, because it makes finding the problem much easier.

Reply:
I would, yes. You can write this section so it has clear directives about what is required for proper support. The key to remember is that folks scan docs, they don't really read them. If they come to a troubleshooting page and the first item they see is "contact support", that's going to be where most readers stop.